applications/virtual-fly-brain/client/src/components/StackViewerComponent.js

```diff
@@ -425,11 +435,11 @@ import React from 'react';
                     if (result[j].trim() !== '') {
                       var index = Number(result[j]);
                       if (i !== 0 || index !== 0) { // don't select template
-                        if (index == 0 && !shift) {
-                          if (!isSelected){
+                        if (index == 0 ) {
+                          if (!window.shiftDown){
                             console.log(that.state.label[i] + ' clicked');
```

Comment:
please clear the console.log if done with the development

Reply:
Done